django_app_server_db_server/deployment/group_vars/app_server

```diff
@@ -1,3 +1,4 @@
 app_server_ip: 192.168.2.54
 db_server_ip: 192.168.3.34
-django_app_home: /opt/eclaim_revamp/eclaim
+django_app_home: "{{django_app_home}}"
```

Comment:
Why are you doing it here like this ? Isn't this like assigning a variable a variable ?